components/__tests__/__snapshots__/locale-chooser.js.snap

```diff
@@ -274,21 +265,25 @@ exports[`opened state 1`] = `
           >
             <glamorous(li)>
               <li
-                className="css-l0f4u7"
+                className="css-1uz022"
               >
                 <glamorous(a)
-                  href="about://blank"
+                  aria-label="Change localization to English"
+                  href="about://undefinedblank"
                 >
                   <a
```

Comment:
The suggestion of adding `aria-hidden="true"` on the SVG eliminates the need for `role="presentation"`